engine/buff/buff.cpp

```diff
@@ -829,6 +831,76 @@ buff_t* buff_t::set_duration_multiplier( double multiplier )
   return this;
 }
 
+buff_t* buff_t::set_time_modifier( double modifier )
+{
+  if ( s_data->flags( spell_attribute::SX_IGNORE_FOR_MOD_TIME_RATE ) )
```

Comment:
This check seems like something that should be done when parsing the spell data rather than when calling this function. For highly specific effects like this, I'd expect that explicitly calling this function does what I tell it regardless of spell data.

Reply:
If you want to keep the attribute check, The normal way to do that would be to add a `bool ignore_mod_time_rate` to `buff_t` and set that based on the presence of the attribute; that way developers can easily override it for specific buffs as needed and you would check for that flag here instead of checking the spell data.